app/pages/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import Head from 'next/head'
+import { default as PageTitle } from '../lib/components/PageTitle'
+import Header from '../lib/components/Header'
+import { Api } from '@mui/icons-material'
+import { dateTime } from 'Luxon'
+
+export default function Home(props) {
+  // const [dates, setDates] = useState([] as string[])
+  // const listItems = props.dates.map((date) =>
+  // <li key={dates.toString()}>{dates}</li>
+  // )
+  const [comments, setComments] = useState([] as any)
+  const [listItems, setListItems] = useState([] as any) 
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+
+    // Get data from the form.
+    const data = {
+      name: event.target.name.value,
+      comment: event.target.comment.value,
+    }
+
+    // Send the data to the server in JSON format.
+    const JSONdata = JSON.stringify(data)
+
+    // API endpoint where we send form data.
+    const endpoint = '/api/postComments'
+
+    // Form the request for sending data to the server.
+    const options = {
+      // The method is POST because we are sending data.
+      method: 'POST',
+      // Tell the server we're sending JSON.
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      // Body of the request is the JSON data we created above.
+      body: JSONdata,
+    }
+
+    // Send the form data to our forms API on Vercel and get a response.
+    const response = await fetch(endpoint, options)
```

Comment:
```suggestion
    // Send the form data to our forms API and get a response.
    const response = await fetch(endpoint, options)
```

We aren't hosting on Vercel! https://vercel.com